capa/render/proto/capa.proto

```diff
@@ -90,7 +90,7 @@ message ExportFeature {
 }
 
 message FeatureCounts {
-  Integer file = 1;
+  uint64 file = 1;  // TODO just int here and general int vs uint?!
```

Comment:
I don't see the benefit of `Integer` message yet vs. `int64`.

Reply:
the range of uint64 cannot represent negative numbers, while sint64/int64 cannot represent 1 << 64. so, the `Integer` type is a union type that lets us store both negative numbers and really big numbers.

I had imagined using some code like the following to aid in the translation.
```py

def int_to_pb2(v):
    assert isinstance(v, int)
    if v < -2_147_483_648:
        raise ValueError("underflow")
    if v > 0xFFFFFFFFFFFFFFFF:
        raise ValueError("overflow")

    if v < 0:
        return capa.render.proto.capa_pb2.Integer(i=v)
    else:
        return capa.render.proto.capa_pb2.Integer(u=v)
 
```

I think we'll also need a similar routine for the `Number` type, which is a union over `Integer` and also doubles, since the capa `Number` feature can be either floating point or integer.